yt_dlp/extractor/dorna.py

```diff
@@ -0,0 +1,393 @@
+import functools
+import json
+import re
+import urllib.parse
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    int_or_none,
+    join_nonempty,
+    OnDemandPagedList,
+    parse_qs,
+    strip_or_none,
+    traverse_obj,
+    unified_strdate,
+    unified_timestamp,
+    update_url_query,
+)
+
+
+class DornaIE(InfoExtractor):
+    _LANGS = r'(?:en|it|de|es|fr)'
+    _DORNA_URLS = {
+        'GET_TOKEN': 'https://sso.dorna.com/api/login/token',
+        'AUTHORIZE': 'https://sso.dorna.com/login/authorize',
+        'CHECK_LOGIN': 'https://sso.dorna.com/api/login/v1/me',
+    }
+    _LANG = None
+    _WEBSITE = None
+    _HOST = None
+    _SECURE_HOST = None
```

Comment:
This should be a single `_DOMAIN` var

Reply:
I kinda need both:

`_LOGIN_URL = f'{_SECURE_HOST}/en/user/login?return_to={_HOST}'`

Reply:
You can just use `https://secure.{_DOMAIN}` and `https://www.{_DOMAIN}` instead of _HOST/_SECURE_HOST respectively. Or even make them properties based on the common _DOMAIN